### PR TITLE
chore: June RC3 releases

### PR DESCRIPTION
**Pull request by skjnldsv**
- [x] https://github.com/nextcloud/server/pull/46005
- [x] https://github.com/nextcloud/server/pull/46006
- [x] https://github.com/nextcloud/server/pull/46008

### CODE REVIEW

**Comment by skjnldsv** [comment]
Ready for review @ChristophWurst @nickvergessen @AndyScherzinger @artonge @Altahrim

File: config/major_versions.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
         "minPHP": "8.0"
     },
     "27": {
-        "eol": "2024-06",
+        "eol": "2024-06-26",
```
Comment (skjnldsv) [inline config/major_versions.json:14]: cc @Altahrim @blizzz

Comment (Altahrim) [reply config/major_versions.json:14]: We test the EOL without the day. 
I think it's a kind of revert of change by @blizzz: https://github.com/nextcloud-releases/updater_server/commit/fac2936feb7b320ca57cc851e3626714ab0bd74c

It depends if we want to mark 27 as the last of the NC 27 or have EOL flag only on July 1st.

I believe we should add the EOL date into the `config.php` so the updater can say it's eol from July 1st.

**Review by Altahrim — APPROVED** [review]

File: config/major_versions.json (hunk 1 above)
Comment (skjnldsv) [reply config/major_versions.json:14]: Yeah, but if I `make config`, then it also changes the stable 27 releases (27.1.10)

Comment (skjnldsv) [reply config/major_versions.json:14]:
```diff
diff --git a/config/config.php b/config/config.php
index a16c268..d15878c 100755
--- a/config/config.php
+++ b/config/config.php
@@ -71,7 +71,7 @@ QnK0Bov0chs7kRKuaA/Hew==',
                                'internalVersion' => '27.1.10.2',
                                'downloadUrl' => 'https://download.nextcloud.com/server/releases/nextcloud-27.1.10.zip',
                                'web' => 'https://docs.nextcloud.com/server/27/admin_manual/maintenance/upgrade.html',
-                               'eol' => false,
+                               'eol' => true,
                                'minPHPVersion' => '8.0',
                                'signature' => '0DZOZXph4HTIx1/OhzIkyveNg7SMISdIZbmcjCRSCFrGK6Ut2u62y6N6xc8D6v0v
 brEvs8RG2hb8HxNBLvPJWBgku2FVJ6DSyXfBxm5cN1d2qQ2fL/DWjFl24++X6Yny
@@ -87,7 +87,7 @@ pFiyELE2Ee7Xpds/CTGunQ==',
                                'internalVersion' => '27.1.10.2',
                                'downloadUrl' => 'https://download.nextcloud.com/server/releases/nextcloud-27.1.10.zip',
                                'web' => 'https://docs.nextcloud.com/server/27/admin_manual/maintenance/upgrade.html',
-                               'eol' => false,
+                               'eol' => true,
                                'minPHPVersion' => '8.0',
                                'signature' => '0DZOZXph4HTIx1/OhzIkyveNg7SMISdIZbmcjCRSCFrGK6Ut2u62y6N6xc8D6v0v
 brEvs8RG2hb8HxNBLvPJWBgku2FVJ6DSyXfBxm5cN1d2qQ2fL/DWjFl24++X6Yny
@@ -714,7 +714,7 @@ mwTKMKNrGW5Jl+SHbV+U+A==',
                                'internalVersion' => '27.1.11.2',
                                'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-27.1.11rc3.zip',
                                'web' => 'https://docs.nextcloud.com/server/27/admin_manual/maintenance/upgrade.html',
-                               'eol' => false,
+                               'eol' => true,
                                'minPHPVersion' => '8.0',
                                'signature' => 'kmzpi3fhMm3qOzj59VF+kAcPVq1/apNpu959bCvQqT/lUOWp6hph3JBCDyjT1FYw
 kgHXO5vnO4yiwAhfsiCvS4dn7gvP3yNJH3W/DagKpNj3e4GvgeunM1f15yU/wgda
@@ -730,7 +730,7 @@ FmqJCVXstlHcbZx1GCFThFCPqDiKQ3jQZuxI+C70tC4g3Lzs28pcIbP4F8NwSwCi
                                'internalVersion' => '27.1.11.2',
                                'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-27.1.11rc3.zip',
                                'web' => 'https://docs.nextcloud.com/server/27/admin_manual/maintenance/upgrade.html',
-                               'eol' => false,
+                               'eol' => true,
                                'minPHPVersion' => '8.0',
                                'signature' => 'kmzpi3fhMm3qOzj59VF+kAcPVq1/apNpu959bCvQqT/lUOWp6hph3JBCDyjT1FYw
 kgHXO5vnO4yiwAhfsiCvS4dn7gvP3yNJH3W/DagKpNj3e4GvgeunM1f15yU/wgda
@@ -763,7 +763,7 @@ FmqJCVXstlHcbZx1GCFThFCPqDiKQ3jQZuxI+C70tC4g3Lzs28pcIbP4F8NwSwCi
                '27' => [
                        'downloadUrl' => 'https://download.nextcloud.com/server/daily/latest-stable27.zip',
                        'web' => 'https://docs.nextcloud.com/server/27/admin_manual/maintenance/upgrade.html',
-                       'eol' => false,
+                       'eol' => true,
                        'minPHPVersion' => '8.0',
                ],
                '26' => [
diff --git a/config/major_versions.json b/config/major_versions.json
index f727e2c..f1a8d45 100644
--- a/config/major_versions.json
+++ b/config/major_versions.json
@@ -11,7 +11,7 @@
         "minPHP": "8.0"
     },
     "27": {
-        "eol": "2024-06-26",
+        "eol": "2024-06",
         "minPHP": "8.0"
     },
     "26": {
```

**Pull request merged**